internal/deepdoc/parser/pdf/inference/client_test.go

```
@@ -124,11 +124,11 @@ func TestDeepDocHTTP_OCRDetect(t *testing.T) {
 		if op := r.FormValue("operator"); op != "det" {
 			t.Errorf("operator = %q, want 'det'", op)
 		}
-		// Verify image is JPEG (not PNG).
+		// Verify image is PNG (not JPEG).
 		file, header, _ := r.FormFile("request")
 		defer file.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `file.Close` is not checked

(errcheck)